### PR TITLE
Linked to other docs from the main README

### PR DESCRIPTION
**Pull request by lenardg**
Added links to the main README.md file to point to the other doc files directly using the linking mechanism supported by Github.

Also fixed a small style thingy in the Linux file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,19 @@ On Windows 10 you can either build for win32 or use XMing and run your builds in
 
 For more information about how to build on the various systems, refer to the platform specific docs in the `docs` folder!
 
-# Overview
+# More docs!
+
+Refer to the OS specific docs:
+
+* [Linux users](docs/Linux.md)
+* [Windows users](docs/Windows.md) or if you are [Visual Studio user](docs/VisualStudio.md)
```
Comment (Daft-Freak) [inline README.md]: Should mention that the first option is WSL/MinGW. Also if you replace 'OS specific' with 'platform specific', then you can list Emscripten and 32blit with the others.

Comment (Gadgetoid) [reply README.md]: I wonder if- faced with good options for using Visual Studio- anyone is actually likely to use WSL/MinGW or WSL/XMing (the latter also doesn't support audio with the current mainline WSL)? We should maybe also move that documentation into a section specifically for Windows uers.

**Comment by Gadgetoid** [comment]
Thank you! Will merge this since the Windows docs questions could probably use some wider discussion elsewhere.

File: README.md [hunk 2]
```diff
@@ -22,7 +22,19 @@ On Windows 10 you can either build for win32 or use XMing and run your builds in
...
+* [MacOSX users](docs/MaxOSX.md)
```
Comment (Daft-Freak) [inline README.md]: A bit late, but there's a typo here: "MaxOSX". (Also I think it's called "macOS" now...)

**Pull request merged**